lib/dpl/providers/cloudfoundry.rb

```diff
@@ -25,7 +25,7 @@ class Cloudfoundry < Provider
       opt '--v3',                  'Use the v3 API version to push the application'
       opt '--logout', default: true, internal: true
 
-      cmds install: 'test $(uname) = "Linux" && rel="linux64-binary" || rel="macosx64"; wget "https://cli.run.pivotal.io/stable?release=${rel}&source=github" -qO cf.tgz && tar -zxvf cf.tgz && rm cf.tgz',
+      cmds install: 'test $(uname) = "Linux" && rel="linux64-binary" || rel="macosx64-binary"; curl -L "https://packages.cloudfoundry.org/stable?release=${rel}&version=v7&source=github" | tar -zx && mv cf /usr/local/bin',
```

Comment:
It just occurred to me:

https://packages.cloudfoundry.org/stable, without query parameters, warns us:

```
Invalid 'release' value, please select one of the following releases: debian32, debian64, redhat32, redhat64, macosx64, windows32, windows64, linux32-binary, linux64-binary, macosx64-binary, windows32-exe, windows64-exe
```

I get the feeling that `linux64-binary` would not work with ARM CPUs (which means we should test `$(uname -m)`), and we probably want to block Windows, too.

Neither of these things are show stopper, as they are not universally enforced at the moment, but I thought I'd point that out.

Reply:
Ok! For now I keep the current installation.